docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -260,8 +271,125 @@ Server can set age rating for current playback. Client can access this rating (r
     }
     ```
 
+## Managing playlists on server side
```

Comment:
Global comment: For consistency, please use Title Case for headings.
Suggest rephrasing to:
## Managing Playlists on Server Side